lxl-web/src/routes/(app)/[[lang=lang]]/[fnurgel=fnurgel]/Holdings.svelte

```diff
@@ -151,15 +162,10 @@
 		& .arrow {
 			@apply rotate-90;
 		}
-
-		& .holder-label {
-			white-space: normal;
-		}
 	}
 
 	.holder-label {
-		@apply flex-1 overflow-hidden text-ellipsis whitespace-nowrap;
-		@apply text-base font-medium;
+		@apply line-clamp-2 text-base font-medium;
```

Comment:
Would prefer to remove `.holder-label` and move these styles inline (see above comment). We are trying to move away from using `@apply` (since it really duplicates styles) and it's not really needed here...